_layouts/syllabus.html

```diff
@@ -55,6 +55,10 @@
       <a href="lectures/{{ page.slidesName }}.pdf">[pdf]</a>
       </p>
     {% endif %}
+    {% for oh_plus_plus in page.ohPlusPlus %}
+      <p>OH++ is opportunity to learn topics more in depth in a lab like setting. Here is a link to this week's OH++:</p>
+      <a href="/static/w17/oh-plus-plus/{{ oh_plus_plus }}">OH++</a>
+    {% endfor %}
```

Comment:
I like having more than just a link to the OH++ markdown, but repeating this in every cell seems a bit redundant.
Maybe just _"OH++ Notes [markdown]"_, that way it mirrors the format for lecture slides, _"Slides [pdf]"_